ivy/functional/frontends/torch/nn/functional/dropout_functions.py

```diff
@@ -13,3 +13,15 @@ def dropout(input, p=0.5, training=True, inplace=False):
         ivy.inplace_update(input, ret)
         return input
     return ret
+
+
+@to_ivy_arrays_and_back
+def dropout2d(tensor, prob=0.5, training=True, data_format="NCHW", inplace=False):
```

Comment:
Original `torch` function doesn't have a `data_format` argument as seen [here](https://pytorch.org/docs/stable/generated/torch.nn.functional.dropout2d.html). Can you please remove this?

Reply:
Oh sorry I've missed that, I will remove it, thank you for mentioning it.